script/NestedSignFromJson.s.sol

```diff
@@ -41,9 +41,20 @@ contract NestedSignFromJson is NestedMultisigBuilder, JsonTxBuilderBase {
         internal
         virtual
         override
+    {
+        if (msg.sig == this.approveJson.selector) {
+            console.log("Skipping assertions on the approval call");
+            return;
+        }
+        _nestedPostCheck(accesses, simPayload);
+    }
+
+    function _nestedPostCheck(Vm.AccountAccess[] memory accesses, SimulationPayload memory simPayload)
```

Comment:
Nit, what do you think about `_postCheckLogic` or similar, so it's more clear for the implementer?
```suggestion
    function _postCheckLogic(Vm.AccountAccess[] memory accesses, SimulationPayload memory simPayload)
```

Reply:
I like that the current name makes it clearer-ish that it's only a requirement for Nested ceremonies.

Reply:
Oh! I read "nested" as meaning "nested within the main postcheck function" lol. Ok, this name does make sense then